Documentation/ceph-cluster-crd.md

```diff
@@ -1489,3 +1490,31 @@ data:
 
 Note: if you are using self-signed certificates (not known/approved by a proper CA) you must pass `VAULT_SKIP_VERIFY: true`.
 Communications will remain encrypted but the validity of the certificate will not be verified.
+
+##### Ceph CSI configuration
+
+Ceph-CSI is capable of encrypting PVs requests.
+Rook can configure Ceph-CSI to turn on encryption with Key Management System support.
+For this, you need to add new parameter in your CephCluster CR spec:
+
+```yaml
+security:
+  kms:
+  ...
+  ...
+  csiConnectionDetailsCMName: "rook-ceph-csi-vault"
```

Comment:
instead of restricting to Vault, keep it open for other KMS types, name it `rook-ceph-csi-kms` by default?

Reply:
It's in the vault section, that's why I added the vault mention in the name :)